### PR TITLE
GrayCards class modified getResource and added the testGrayCard

### PR DESCRIPTION
**Pull request by Solrac1906**

### CODE REVIEW

File: src/test/java/jalau/at18/architects/TestGrayCard.java [hunk 1]
```diff
@@ -1,7 +1,10 @@
 package jalau.at18.architects;
 
 import org.junit.Test;
-import org.junit.Assert;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
 public class TestGrayCard {
```
Comment (J0c4) [inline src/test/java/jalau/at18/architects/TestGrayCard.java:8]: In order to keep consistency with other test files, please rename this class to `GrayCardTest`

**Review by J0c4 — CHANGES_REQUESTED** [review]
Looks good, please check my comment. Also, this needs to be rebased, please do that as well

File: src/test/java/jalau/at18/architects/TestGrayCard.java (hunk 1 above)
Comment (J0c4) [reply src/test/java/jalau/at18/architects/TestGrayCard.java:8]: Hello @Solrac1906 , any updates on this one?

Comment (Solrac1906) [reply src/test/java/jalau/at18/architects/TestGrayCard.java]: fixed the observations

**Review by J0c4 — APPROVED** [review]

**Pull request merged**